chef/cookbooks/provisioner/templates/default/autoyast-upgrade.xml.erb

```diff
@@ -91,6 +91,25 @@
         </source>
       </script>
     </chroot-scripts>
+    <init-scripts config:type="list">
+      <!-- bugfix bnc#886238: https://bugzilla.novell.com/show_bug.cgi?id=886238 -->
+      <script>
+       <filename>autoyast_set_hostentries.sh</filename>
```

Comment:
Please remove this one. We don't need that for the upgrade. It would create duplicate entries in /etc/hosts  I think.